src/ServiceProvider/Kernel/ServiceProviderSpecimenBuilder.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using AutoFixture.Kernel;
+
+namespace Kralizek.AutoFixture.Extensions.Kernel;
+
+public class ServiceProviderSpecimenBuilder : ISpecimenBuilder
+{
+    private readonly IServiceProvider _serviceProvider;
+
+    public ServiceProviderSpecimenBuilder(IServiceProvider serviceProvider)
+    {
+        _serviceProvider = serviceProvider ?? throw new ArgumentNullException(nameof(serviceProvider));
+    }
+    
+    public object Create(object request, ISpecimenContext context)
+    {
+        _ = request ?? throw new ArgumentNullException(nameof(request));
+
+        _ = context ?? throw new ArgumentNullException(nameof(context));
```

Comment:
The `context` is not used in this method, so I assume that you don't really care if it's null or not.

Reply:
good point.